crates/extensions/c8y_mapper_ext/src/dynamic_discovery.rs

```diff
@@ -40,6 +42,11 @@ pub fn process_inotify_events(
         .parent()
         .ok_or_else(|| DynamicDiscoverOpsError::NotAnOperation(path.to_path_buf()))?;
 
+    // only files inside `/etc/tedge/operations/c8y` directory are valid c8y operations
```

Comment:
The watcher is watching `/etc/tedge/operations/c8y` directory itself as well, so if there are events related to the directory, we get a `/etc/tedge/operations` parent and `c8y` as filename, which is wrong. It may have changed when I upgraded the fs notification library for the debouncing PR. I think it's good to not rely that fsnotify will give us proper paths and have this guard here just to be sure.